va/va_enc_mpeg2.h

```diff
@@ -287,6 +287,17 @@ typedef struct _VAEncSliceParameterBufferMPEG2 {
     uint32_t                va_reserved[VA_PADDING_LOW];
 } VAEncSliceParameterBufferMPEG2;
 
+typedef struct _VAEncMiscParameterExtensionDataSeqDisplayMPEG2
```

Comment:
Is it the responsibility of the driver to include the Packed header? Or does the middleware supposed to insert them? In any case, we should comment it.

Reply:
application query whether it driver support packed header insertion by VAConfigAttribEncPackedHeaders.  if not support , driver will packed the header. this information is used for pack header inside driver